lib/ex_plasma/utils/rlp_decoder.ex

```diff
@@ -0,0 +1,26 @@
+defmodule ExPlasma.Utils.RlpDecoder do
+  @moduledoc """
+  Provides functions to decode various data types from RLP raw format
+  """
+
+  @doc """
+  Attempt to decode the given binary to a list of RLP items
+  """
+  @spec decode(binary()) :: {:ok, list()} | {:error, :malformed_rlp}
+  def decode(tx_bytes) do
+    {:ok, ExRLP.decode(tx_bytes)}
+  rescue
+    _ -> {:error, :malformed_rlp}
```

Comment:
same as the above `rescue` comment. it would be better if we catch specific errors